lte/gateway/c/connection_tracker/EventTracker.cpp

```diff
@@ -0,0 +1,246 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "EventTracker.h"
+
+static int parse_ip_cb(const struct nlattr* attr, void* data) {
```

Comment:
Can we move these helper functions to a separate place? Maybe `EventTrackerHelpers` and then include that here, ideally this source file should only contain the definition of the `EventTracker` member class functions

Reply:
these are local static helper functions, isn't the best practice to just keep in the file that uses it here? I'm open to moving them though, should I just stil them into EventTrackerHelpers.h?